deploy/kubernetes/deployment.yaml

```diff
@@ -33,7 +33,7 @@ spec:
       serviceAccount: csi-attacher
       containers:
         - name: csi-attacher
-          image: gcr.io/k8s-staging-sig-storage/csi-attacher:canary
+          image: registry.k8s.io/k8s-staging-sig-storage/csi-attacher:canary
```

Comment:
Use `registry.k8s.io/sig-storage/csi-attacher:v3.5.0`. I don't see `:canary` builds in registry.k8s.io.

Reply:
Ok, I didn't check that, will update this PR